filament/backend/include/private/backend/CommandStream.h

```diff
@@ -34,6 +34,7 @@
 #include <utils/compiler.h>
 #include <utils/ThreadUtils.h>
 
+#include <cstddef>
```

Comment:
use `<stddef.h>`

Reply:
No... It's because of 
`error: no member named 'max_align_t' in namespace 'std'`

- it seems **max_align_t** is **ONLY** in <cstddef>,
- and <stddef.h> is **already** included in the next couple lf lines in the same file...

Reply:
okay, sounds good then. Sorry about the confusion.